From 8c7ca597bcb15110d4b33a00eee426acf34c6d29 Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Wed, 5 Apr 2017 18:10:54 +0200
Subject: [PATCH 2/8] nxp: jni: Forward-port the stack sources

Author: Adrian DC <radian.dc@gmail.com>
Date:   Wed Apr 5 18:10:54 2017 +0200

    Nfc: Include android-support-v4 for the NXP stack

     * Introduced for the regular package compilation in
        "Use FileProvider rather than sharing raw file URI"
        I28befb70743ae84825b4459d1433e8982b0a74ac

    Change-Id: Ibbfe6e5b6c9657ab7c04bd02982528fb2f941192

Author: Hiroki Yamamoto <hiroki.a.yamamoto@sony.com>
Date:   Wed Apr 12 17:53:28 2017 +0900

    nxp: liblog: replace android/log.h to log/log.h

    To access main logging macros

    Test: compile
    Change-Id: I349928d348c87409ddcd90c8073e2517119553f2
    Signed-off-by: Adrian DC <radian.dc@gmail.com>

Change-Id: Id13ad5203bdc5c4f186d9e3ca4c45aacb4c78b5e
---
 Android.mk                       | 2 ++
 nxp/jni/com_android_nfc.h        | 2 +-
 nxp/jni/com_android_nfc_list.cpp | 2 +-
 3 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index fd299edc..13a1f95c 100644
--- a/Android.mk
+++ b/Android.mk
@@ -20,6 +20,8 @@ LOCAL_CERTIFICATE := platform
 
 LOCAL_JNI_SHARED_LIBRARIES  := libnfc_jni
 
+LOCAL_STATIC_JAVA_LIBRARIES := android-support-v4
+
 LOCAL_PROGUARD_ENABLED := disabled
 
 include $(BUILD_PACKAGE)
diff --git a/nxp/jni/com_android_nfc.h b/nxp/jni/com_android_nfc.h
index 99ad0326..67fa5815 100644
--- a/nxp/jni/com_android_nfc.h
+++ b/nxp/jni/com_android_nfc.h
@@ -24,7 +24,7 @@
 #include <semaphore.h>
 #include <sys/queue.h>
 
-#include <android/log.h>
+#include <log/log.h>
 #include <cutils/properties.h> // for property_get
 
 #include <JNIHelp.h>
diff --git a/nxp/jni/com_android_nfc_list.cpp b/nxp/jni/com_android_nfc_list.cpp
index 749417c0..c41e6dfa 100644
--- a/nxp/jni/com_android_nfc_list.cpp
+++ b/nxp/jni/com_android_nfc_list.cpp
@@ -24,7 +24,7 @@
 #include <malloc.h>
 #include <pthread.h>
 
-#include <android/log.h>
+#include <log/log.h>
 
 bool listInit(listHead* pList)
 {
-- 
2.17.1

